README.md

```diff
@@ -1 +1,81 @@
 # Internet Button Playground
+
+The repository contains code used to create [Internet Button](https://store.particle.io/products/internet-button) projects.
+
+## Projects
+
```

Comment:
In a follow up, let's document https://github.com/stevector/internet-button-playground/pull/16